testsuite/integration/elytron-oidc-client/src/test/java/org/wildfly/test/integration/elytron/oidc/client/KeycloakContainer.java

```diff
@@ -67,6 +67,7 @@ protected void configure() {
             waitingFor(Wait.forHttp("/").forPort(PORT_HTTP));
             withCommand("start-dev");
         }
+        waitingFor(Wait.forLogMessage(".*Keycloak.*started.*", 1));
```

Comment:
Just realized, I think this line will need to be moved into the `else` block above since I think the message for Keycloak is different from the message for RH SSO.

Reply:
Ah, that would make sense. I'll make that change and push up the commit again with some added logging since it still seems to be failing on CI, but I can't get it to fail locally.